website/project/model.py

```diff
@@ -4092,15 +4094,28 @@ def _email_template_context(self, user, is_authorizer=False, urls=None):
 
     def _add_success_logs(self, node, user):
         src = node.registered_from
+        submitted_time = ''
+
+        try:
+            draft = DraftRegistration.find_one(Q("registered_node", "eq", node))
+            # lazy log choosing [value if false, value if true][<test - truthy value]()
+            log_type = [NodeLog.PROJECT_REGISTERED, NodeLog.PREREG_CHALLENGE][
+                "Prereg" in draft.registration_schema.name]
```

Comment:
I posted about this [on the commit](https://github.com/Marcus-Rosti/osf.io/commit/21d6fc2e1b77c6c16158c7399a85b994047d2815#commitcomment-15988939). Summarizing on the PR diff for easier linking ... let's avoid the overly clever construction here. Three fine options would be:

```python
is_prereg = "Prereg" in draft.registration_schema.name
log_type = NodeLog.PREREG_CHALLENGE if is_prereg else NodeLog.PROJECT_REGISTERED
```

--------------

```python
if "Prereg" in draft.registration_schema.name:
    log_type = NodeLog.PREREG_CHALLENGE
else:
    log_type = NodeLog.PROJECT_REGISTERED
```

---------

```python
log_type = NodeLog.PROJECT_REGISTERED
if "Prereg" in draft.registration_schema.name:
    log_type = NodeLog.PREREG_CHALLENGE
```